include/functions.h

```diff
@@ -1135,41 +1135,41 @@ Gfx* Gfx_SetFog(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_SetFogWithSync(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_SetFog2(Gfx* gfx, s32 r, s32 g, s32 b, s32 a, s32 near, s32 far);
 Gfx* Gfx_CallSetupDL(Gfx* gfx, u32 i);
-Gfx* func_800937C0(Gfx* gfx);
-Gfx* func_80093808(Gfx* gfx);
-void func_800938B4(GraphicsContext* gfxCtx);
-void func_8009398C(GraphicsContext* gfxCtx);
-void func_80093AD0(GraphicsContext* gfxCtx);
-void func_80093BA8(GraphicsContext* gfxCtx);
-void func_80093C14(GraphicsContext* gfxCtx);
+Gfx* Gfx_SetupDL57(Gfx* gfx);
+Gfx* Gfx_SetupDL52NoCD(Gfx* gfx);
+void Gfx_SetupDL57Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL51Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL54Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL26Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL25Xlu2(GraphicsContext* gfxCtx);
 void func_80093C80(GlobalContext* globalCtx);
-void func_80093D18(GraphicsContext* gfxCtx);
-void func_80093D84(GraphicsContext* gfxCtx);
-Gfx* func_80093F34(Gfx* gfx);
-Gfx* func_80093F58(Gfx* gfx);
-void func_80094044(GraphicsContext* gfxCtx);
-void func_800940B0(GraphicsContext* gfxCtx);
-void func_80094140(GraphicsContext* gfxCtx);
-Gfx* func_8009411C(Gfx* gfx);
-void func_800942F0(GraphicsContext* gfxCtx);
-void func_8009435C(GraphicsContext* gfxCtx);
-void func_800943C8(GraphicsContext* gfxCtx);
-Gfx* func_800944A0(Gfx* gfx);
-void func_800944C4(GraphicsContext* gfxCtx);
-void func_80094520(GraphicsContext* gfxCtx);
-void func_8009457C(Gfx** gfxp);
-void func_800945A0(GraphicsContext* gfxCtx);
-void func_8009460C(GraphicsContext* gfxCtx);
-void func_80094678(GraphicsContext* gfxCtx);
-Gfx* func_80094968(Gfx* gfx);
-Gfx* func_800946E4(Gfx* gfx);
+void Gfx_SetupDL25Opa(GraphicsContext* gfxCtx);
+void Gfx_SetupDL25Xlu(GraphicsContext* gfxCtx);
+Gfx* Gfx_SetupDL64(Gfx* gfx);
+Gfx* Gfx_SetupDL34(Gfx* gfx);
+void Gfx_SetupDL44Xlu(GraphicsContext* gfxCtx);
```

Comment:
Usually I dont like adding extra underscores in function names, but Im kinda thinking it might be warranted for this. 
`Gfx_SetupDL37Opa` vs `Gfx_SetupDL_37Opa`, for example

Trying to make out where the name of the function ends and the numbers start feels hard at times to me.
I guess it depends on if we intend to keep these names with the numbers, or if these are temporary.

Reply:
As far as I'm concerned, we are going to keep these numbered names, since the systematic naming I suggested as a last resort wasn't convincing.
We may name a few by usage eventually (?) but definitely not all of them

Reply:
I think having that second underscore helps with the clarify of it in this particular case, since it's a lot of (common) abbreviations in a short amount of space, so it gets quite scrambled on first glance